### PR TITLE
indexer-integration-tests: start integration test package

### PR DESCRIPTION
**Pull request by dwerner**
Small PR to add the integration test package to the lerna workspace.

### CODE REVIEW

File: packages/indexer-integration-tests/package.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "indexer-integration-tests",
+  "version": "0.0.1",
+  "description": "Integration tests for the indexer components.",
+  "main": "index.js",
+  "repository": "https://github.com/graphprotocol/indexer",
+  "author": "Graph Protocol",
+  "scripts": {
+    "integration-test": "ts-node src/bootstrapTest.ts"
```
Comment (fordN) [inline packages/indexer-integration-tests/package.json:9]: I know this is basically just a placeholder for now, but I'd expect a `jest ...` command here..?

File: packages/indexer-integration-tests/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "declaration": true,
+    "sourceMap": true,
+    "esModuleInterop": true,
+    "strict": true,
+    "composite": true,
+    "lib": ["es2015", "es6", "esnext.asynciterable", "dom", "ES2020.Promise"]
+  },
+  "include": ["tests/**/*.ts", "tests/**/*.d.ts"],
+  "exclude": ["tests/**/__tests__/*.ts"],
+  "references": [{ "path": "../indexer-common" }]
```
Comment (fordN) [inline packages/indexer-integration-tests/tsconfig.json:17]: May want to include references to other indexer packages too.

**Review by fordN — APPROVED** [review]
👍 

Left a few little questions and comments. Feel free to merge though; I feel they will naturally be resolved in next steps of actually setting up tests anyways.

File: packages/indexer-integration-tests/package.json (hunk 1 above)
Comment (dwerner) [reply packages/indexer-integration-tests/package.json:9]: Ah, yeah leftover of me figuring this out. Will remove.

File: packages/indexer-integration-tests/tsconfig.json (hunk 2 above)
Comment (dwerner) [reply packages/indexer-integration-tests/tsconfig.json:17]: This is copy-paste from another package, but yes definitely. I'll leave this as is to remind about that potential.

**Pull request merged**